form/storage/storage_reader.cpp

```
@@ -6,8 +6,189 @@
+    auto const per_file = per_tech->second.find(file_name);
+    if (per_file == per_tech->second.end()) {
+      return {};
+    }
+    return per_file->second;
+  }
+
+  form::experimental::config::tech_setting_config::table_t get_container_table(
+    form::experimental::config::tech_setting_config const& settings,
+    int technology,
+    std::string const& container_name)
+  {
```

Cppcheck (2.21.0)

[style] The function 'getItems' is never used.

(unusedFunction)